gapic/src/main/com/google/gapid/widgets/TabComposite.java

```diff
@@ -20,6 +20,7 @@
 import com.google.gapid.models.Analytics;
 import com.google.gapid.models.Analytics.View;
 import com.google.gapid.util.Events;
+import com.google.gapid.views.TabContent;
```

Comment:
Yikes. This is a dependency between packages I don't like. I think the solution is to:

1) keep the original `views.Tab` interface with `reinitialize` method
2) Create the new content interface as an inner class here.
3) make `views.Tab` extend the new content interface.

Reply:
Sure, done.